doc/operator-application-capabilities.md

```diff
@@ -629,6 +636,116 @@ oc create secret generic threescale-provider-account --from-literal=adminURL=htt
 
 The operator will gather required credentials automatically for the default 3scale tenant (provider account) if 3scale installation is found in the same namespace as the custom resource.
 
+## OpenAPI custom resource
+
+### Features
+
+* [OpenAPI 3.0.2](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md) specification
+  * Just a single top level security requirement supported. Operation level security requirements not supported.
+  * Only first `server.url` element in servers list parsed as private url. Path and operation level `servers` elements not supported.
+  * Supported security schemes: `apiKey`.
+* OpenAPI spec document can be read from:
+  * ConfigMap
+  * URL. Supported schemes are http and https
```

Comment:
Can we document accepted formats of the openapi spec (for both url and configmap)? Is it json, yaml, or both?
(same info in openapi crd doc would be useful too)